src-gui/src/renderer/components/modal/feedback/FeedbackDialog.tsx

```diff
@@ -1,356 +1,222 @@
-        <ul>
-          <li>Got something to say? Drop us a message below. </li>
-          <li>If you had an issue with a specific swap, select it from the dropdown to attach the logs.
-            It will help us figure out what went wrong.
-          </li>
-          <li>We appreciate you taking the time to share your thoughts! Every message is read by a core developer!</li>
-        </ul>
-        <Box
-          style={{
-            display: "flex",
-            flexDirection: "column",
-            gap: "1rem",
-          }}
-        >
-          <TextField
-            variant="outlined"
-            value={bodyText}
-            onChange={(e) => setBodyText(e.target.value)}
-            label={
-              bodyTooLong
-                ? `Text is too long (${bodyText.length}/${MAX_FEEDBACK_LENGTH})`
-                : "Message"
-            }
-            multiline
-            minRows={4}
-            maxRows={4}
-            fullWidth
-            error={bodyTooLong}
-          />
-          <Box style={{
-            display: "flex",
-            flexDirection: "row",
-            justifyContent: "space-between",
-            gap: "1rem",
-          }}>
-
-            <SwapSelectDropDown
-              selectedSwap={selectedSwap}
-              setSelectedSwap={setSelectedSwap}
-            />
-            <Tooltip title="View the logs">
-              <Box style={{ display: "flex", alignItems: "center", justifyContent: "center" }}>
-                <IconButton onClick={() => setSwapLogsEditorOpen(true)} disabled={selectedSwap === null}>
-                  <Visibility />
-                </IconButton>
-              </Box>
-            </Tooltip>
-          </Box>
-          <LogViewer open={swapLogsEditorOpen} setOpen={setSwapLogsEditorOpen} logs={swapLogs} redact={setSwapLogsRedacted} />
-          <Box style={{
-            display: "flex",
-            flexDirection: "row",
-            justifyContent: "space-between",
-            gap: "1rem",
-          }}>
-            <Paper variant="outlined" style={{ padding: "0.5rem", width: "100%" }} >
-              <FormControlLabel
-                control={
-                  <Checkbox
+    const bodyTooLong = input.bodyText.length > MAX_FEEDBACK_LENGTH
+
+    return (
+        <Dialog open={open} onClose={handleClose}>
+            <DialogTitle style={{ paddingBottom: '0.5rem' }}>
+                Submit Feedback
+            </DialogTitle>
+            <DialogContent>
+                <Box
+                    style={{
+                        display: 'flex',
+                        flexDirection: 'column',
+                        gap: '1.5rem',
+                    }}
+                >
+                    <Box>
+                        <Typography style={{ marginBottom: '0.5rem' }}>
+                            Have a question or need assistance? Message us below
+                            or <ExternalLink href="mailto:help@unstoppableswap.net">email us</ExternalLink>!
```

Comment:
I'd make this copyable maybe? Clicking the link doesn't for me.

You can use a `ActionableMonospaceTextBox`